fuzzbucket_client/__main__.py

```diff
@@ -1100,7 +1130,7 @@ def _urlopen(self, request):
             yield response
 
     @property
-    def _url(self):
+    def _url(self) -> typing.Optional[str]:
         return self._env.get("FUZZBUCKET_URL")
```

Comment:
Worth returning a default of empty string so you don't have all the string coercion going on?

Reply:
I like that better, yep 👍🏼